app/src/test/kotlin/io/eugenethedev/taigamobile/viewmodels/BaseViewModelTest.kt

```diff
@@ -2,6 +2,7 @@ package io.eugenethedev.taigamobile.viewmodels
 
 import android.content.Context
 import io.eugenethedev.taigamobile.dagger.AppComponent
+import io.eugenethedev.taigamobile.dagger.DataModule
 import io.eugenethedev.taigamobile.domain.repositories.*
 import io.eugenethedev.taigamobile.state.Session
 import io.eugenethedev.taigamobile.state.Settings
```

Comment:
Look at `import kotlinx.coroutines.test.UnconfinedTestDispatcher` - it can't be found